<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NetTransfer</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>540</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Peteck Develops</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Protocolos</string>
    </property>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>TCP</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>UDP</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Checagem</string>
    </property>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>CRC</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Checksum">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Checksum</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>331</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>331</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cliente</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>289</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Servidor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enviar para o host</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enviar para IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>540</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>540</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CRC/CHECKSUM</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>540</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CRC/CHECKSUM</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuNetConnect_Sofware">
    <property name="title">
     <string>Modo de arquivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionAudio"/>
    <addaction name="actionDatai"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionBefehle"/>
   </widget>
   <addaction name="menuNetConnect_Sofware"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAudio">
   <property name="text">
    <string>imagem</string>
   </property>
  </action>
  <action name="actionDatai">
   <property name="text">
    <string>Data</string>
   </property>
  </action>
  <action name="actionBefehle">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
